# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../netsniff-ng/0001-include-sys_types.h.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./link.c.orig	2014-04-07 14:39:56.297637200 +0200
+++ ./link.c	2014-04-07 14:40:16.478105452 +0200
@@ -1,3 +1,4 @@
+#include <sys/types.h>	/* caddr_t */
 #include <sys/ioctl.h>
 #include <linux/sockios.h>
 #include <unistd.h>
